Portfolio.html

```diff
@@ -3,12 +3,86 @@
 <head>
```

Comment:
lowercase filenames please

Reply:
also this should be index.html - so it works on the server

Reply:
done.